### PR TITLE
Update start-mining.md

### PR DESCRIPTION
**Pull request by dantrevino**
toml file from krypton is confusing users. updated to the same as https://raw.githubusercontent.com/blockstack/stacks-blockchain/master/testnet/stacks-node/conf/xenon-miner-conf.toml

## Description

Describe the changes that were made in this pull request. When possible start with the motivations behind the change. Be sure to also include the following information:

1. Motivation for change?
Documentation is wrong for Xenon.
2. What was changed?
Miner toml file in the example
3. Link to relevant issues?
discord complaints

## Checklist

- [ ] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] New links to files and images were verified
- [ ] For fixes/refactors: all existing references were identified and replaced
- [ ] [Style guide](https://developers.google.com/style) was reviewed and applied
- [ ] Clear code samples were provided
- [ ] People were tagged for review

### CODE REVIEW

**Review by lgalabru — COMMENTED** [review]
Looks good to me, thank you @dantrevino

**Review by hstove — COMMENTED** [review]
Looks good!

**Comment by hstove** [comment]
Did this get merged? Or just closed?

**Comment by dantrevino** [comment]
@hstove do you know how the updated docs get published?  I'm still seeing the old data.

File: src/pages/start-mining.md [hunk 1]
```diff
@@ -98,33 +98,32 @@ Paste in the following configuration:
...
-# Details can be found in above section, 'Running bitcoind locally'
-# For Krypton, set peer_host to `bitcoind.krypton.blockstack.org` and
-# omit `username` and `password`
+mode = "xenon"
 peer_host = "127.0.0.1"
-username = "your-bitcoind-username"
-password = "your-bitcoind-password"
 rpc_port = 18332
 peer_port = 18333
+username = "xenon"
```
Comment (friedger) [inline src/pages/start-mining.md:112]: your-bitcoind-username makes more sense here.

File: src/pages/start-mining.md [hunk 2]
```diff
@@ -98,33 +98,32 @@ Paste in the following configuration:
...
+password = "password"
```
Comment (friedger) [inline src/pages/start-mining.md:113]: Same here: your-bitcoind-password

**Review by friedger — APPROVED** [review]
The rpc changes should go into a different PR.

**Comment by agraebe** [comment]
@CharlieC3 and @wileyj should review as well. Given that this is a tutorial, I'm assuming some content updates are needed to clarify the new fields

**Comment by dantrevino** [comment]
@agraebe @friedger resolved the rpc username/password callout as well as added back the "mode" comments

**Comment by dantrevino** [comment]
replaced by #1005

**Pull request closed without merging**